mr/src/test/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandlerBaseTest.java

```diff
@@ -157,6 +177,282 @@ public void testJoinTables() throws IOException {
     Assert.assertArrayEquals(new Object[] {1L, "Bob", 102L, 33.33d}, rows.get(2));
   }
 
+  @Test
+  public void testCreateDropTable() throws TException, IOException {
+    // We need the location for HadoopTable based tests only
+    String location = locationForCreateTable(temp.getRoot().getPath(), "customers");
+    shell.executeStatement("CREATE EXTERNAL TABLE customers " +
+        "STORED BY 'org.apache.iceberg.mr.hive.HiveIcebergStorageHandler' " +
+        (location != null ? "LOCATION '" + location + "' " : "") +
+        "TBLPROPERTIES ('" + InputFormatConfig.TABLE_SCHEMA + "'='" + SchemaParser.toJson(CUSTOMER_SCHEMA) + "', " +
+        "'" + InputFormatConfig.PARTITION_SPEC + "'='" + PartitionSpecParser.toJson(IDENTITY_SPEC) + "', " +
+        "'dummy'='test')");
+
+    Properties properties = new Properties();
+    properties.put(Catalogs.NAME, TableIdentifier.of("default", "customers").toString());
+    if (location != null) {
+      properties.put(Catalogs.LOCATION, location);
+    }
+
+    // Check the Iceberg table data
+    org.apache.iceberg.Table icebergTable = Catalogs.loadTable(shell.getHiveConf(), properties);
+    Assert.assertEquals(SchemaParser.toJson(CUSTOMER_SCHEMA), SchemaParser.toJson(icebergTable.schema()));
```

Comment:
Instead of comparing the strings, it would be better to compare `icebergTable.schema().asStruct()`. `StructType` implements `equals`, but `Schema` doesn't because it may have extra metadata (like aliases).

Reply:
Done